src/router/index.js

```diff
@@ -47,6 +47,11 @@ export default new Router({
 			path: '/',
 			component: Timeline,
 			name: 'root',
+			props: route => ({
+				path: `/${route.params.path ? route.params.path : ''}`,
```

Comment:
There is no path in the timeline :)
So it doesn't make sense, also isRoot should be always true then ?

Reply:
True!